tools/topology/topology2/include/pipelines/cavs/passthrough-capture.conf

```diff
@@ -16,9 +16,9 @@
 # Where N is the unique pipeline ID within the same alsaconf node.
 #
 
-<include/common/audio_format.conf>
 <include/components/copier.conf>
 <include/components/pipeline.conf>
+<include/common/audio_format.conf>
```

Comment:
is this an unrelated change? the original order is in alphabetical order no?

Reply:
it's to align passthrough-capture and passthrough-playback. One of them doesn't follow the convention. I don't care what the convention is, I just want the two files to be aligned.